etcdserver/server.go

```diff
@@ -1988,6 +1992,67 @@ func (s *EtcdServer) sync(timeout time.Duration) {
 	})
 }
 
+// publishV3 registers server information into the cluster using v3 request. The
+// information is the JSON representation of this server's member struct, updated
+// with the static clientURLs of the server.
+// The function keeps attempting to register until it succeeds,
+// or its server is stopped.
```

Comment:
Add a TODO to replace publish() in etcd 3.6?

Reply:
Added